src/entities/shapeEntity.js

```diff
@@ -48,7 +49,15 @@ function ShapeEntity(Interceptor, shapeObject, shapeArgs, canvasX, canvasY) {
         }
       }
     } else if (!objectType.localeCompare(`ellipse`)) {
-      objectArea = 3.14 * shapeArgs[2] * shapeArgs[3] / 4;
+      if(!this.currentEllipseMode.localeCompare(`center`)) {
```

Comment:
it seems like currentEllipseMode is not yet declared. Could you check and make sure everything is fine in the src/entities/ellipseModeEntity.js